### DISCUSSION TITLE
Your account has been Flagged

### DISCUSSION THREAD

**Discussion by Muzammil-E** [community/community#32160] [New to GitHub] [answered]
So I signed up for github only to realize that I misspelled my primary email. So I add the correct email as the secondary and delete the misspelled one. Then I verify my account and enter my launch code. That is when my account got flagged. How can I get the flag removed asap?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact).

**Reply by Muzammil-E** [reply]
thank you!

**Comment by d1i2v3y4a** [comment]
sir my account was flagged plz activativated

**Reply by queenofcorgis** [reply]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as we cannot assist in flagged account matters :)
